lib/tasks/unpublish.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+namespace :unpublish do
+  desc "Retire a document"
+  task :retire_document, %I(base_path explanatory_note) => :environment do |_, args|
+    raise "Missing base_path parameter" unless args.base_path
+    raise "Missing explanatory_note parameter" unless args.explanatory_note
+
+    document = Document.find_by(base_path: args.base_path)
```

Comment:
I'm not sure we can safely rely on base path here to remove documents as it could have changed in a later draft of their document. 

I'd probably suggest using the `fe7c00c1-a34f-419c-88eb-c844ba70b1c5:en` hybrid since that's the identifier we use for all our URLs?

Reply:
I'll add locale as a optional parameter to the rake task, with "en" as the default and make "content_id" the required param rather than base_path.